raiden_contracts/data/gas.json

```diff
@@ -3,32 +3,32 @@
     "CustomToken.mint": 50776,
     "CustomToken.transfer": 36556,
     "CustomToken.transferFrom": 29060,
-    "MonitoringService.claimReward": 45555,
-    "MonitoringService.monitor": 224140,
-    "OneToN.bulkClaim 1 ious": 106874,
-    "OneToN.bulkClaim 6 ious": 165202,
-    "OneToN.claim": 103516,
-    "SecretRegistry.registerSecret": 46205,
+    "MonitoringService.claimReward": 45566,
```

Comment:
Not an actual review comment. But I'm a little confused why we became more expensive. Shouldn't it stay basically equal? 🤔 
Though the relative change is so small it probably doesn't matter. Just a self-reflection of what has changed that affects gas usage.

Reply:
This is automatically updated with the command `make update_gas_costs`.